### DISCUSSION TITLE
Undocumented breaking change? (`stripe_checkout_session_id` vs `session_id`)

### DISCUSSION THREAD

**Discussion by marckohlbrugge** [pay-rails/pay#1110] [Q&A]
https://github.com/pay-rails/pay/commit/40c637fda8f1f03ac3ee1fae6f7a0a99806d67c0 changed the default success URL parameter from `session_id` to `stripe_checkout_session_id`.

This broke my Pay implementation which had code like this:

```ruby
Pay::Stripe.sync_checkout_session(params[:session_id])
```

Which no longer works, because Pay uses `stripe_checkout_session_id`.

The fix is easy. Just switch to using `params[:stripe_checkout_session_id]`, or better yet use `Pay.sync(params)`.

But I was surprised to run into this breaking change which doesn't seem to be documented in the changelog?
I suggest adding it retroactively.

**Comment by excid3** [comment]
Yeah, forgot to add that. If you use the `pay_customer.checkout` helper, it will automatically set that param, so it's not something you need to change in that case.
